compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/TypeChecker.java

```diff
@@ -8162,16 +8175,18 @@ private boolean couldHoldTableValues(BType type, List<BType> encounteredTypes) {
     }
 
     private boolean isConst(BLangExpression expression) {
-
-        if (ConstantAnalyzer.isValidConstantExpressionNode(expression)) {
-            return true;
-        }
-
-        if (expression.getKind() != NodeKind.SIMPLE_VARIABLE_REF) {
-            return false;
+        switch (expression.getKind()) {
+            case LITERAL:
```

Comment:
Are these codecov warnings false negatives?

Reply:
Yes :-)

Reply:
Shall we sync the PR with master and rerun the build also?